app/templates/help_desk_base.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+
+{% block html_attribs %}
+    <html lang="en">
+{% endblock %}
```

Comment:
You shouldn't need to set blocks, if you aren't intending on replacing them elsewhere. This might apply to other spots